### DISCUSSION TITLE
Local plyr.svg file

### DISCUSSION THREAD

**Discussion by gnudarve** [sampotts/plyr#2470] [Q&A]
We need plyr.svg to be local for security purposes, we are trying to reference a plyr.svg file located in the same folder as plyr.js.  Tried using:        

const player = new Plyr('.js-player', {
            "controls": ['rewind', 'play', 'fast-forward', 'progress', 'current-time', 'duration', 'volume'],
            "iconUrl": 'file://plyr.svg',
            "blankVideo": 'file://blank.mp4'
        }
   );

But it does not work.  Tried many ways to reference it but I can't seem to find the right syntax.  Do you know how would you reference a local file for iconUrl and blankVideo?

**Comment by henriquecustodia** [comment]
Try to use a http server to deliver assets instead file protol
